src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -1263,6 +1263,17 @@ public function get_tag() {
 		}
 	}
 
+	/**
+	 * Return how deep the currently-matched element is in the HTML document.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @return int The depth of the currently-matched element.
```

Comment:
Should we be talking about "element" here? It doesn't need to be an element we're at… should this talk about "token" or more generally "position of the HTML processor"?

Reply:
I think we should say "node". These can be `#text` nodes here. Is "position" right? I think in places we've used "location in the document" but also in the HTML Processor I feel like we're free to talk more specifically about nodes and elements since that's what it operates on, vs. text tokens.

Reply:
Node sounds good.

Reply:
my own confusing is clearer now: I had un-pushed work, including the method itself. I've sent this out in d54d71fc94

apparently I had already chosen "of the current location" in my work, so I'm leaving it at that unless we prefer "node"